components/webgpu/lib.rs

```diff
@@ -82,33 +82,33 @@ pub enum WebGPURequest {
     },
     CopyBufferToBuffer {
```

Comment:
nit: might be good to move them together as `Copy { command_encoder_id: id::CommandEncoderId, op: Option<CopyOperation> }` or something like this. That would allow the server side to handle the errors more uniformly.